lib/clients/authenticated.js

```diff
@@ -111,17 +111,24 @@ _.assign(AuthenticatedClient.prototype, new function() {
   prototype._placeOrder = function(params, callback) {
     var self = this;
 
-    var requiredParams = ['size', 'side', 'product_id'];
+    var requiredParams = ['side', 'product_id'];
 
     if (params.type !== 'market') {
       requiredParams.push('price');
+      requiredParams.push('size');
```

Comment:
👍 I was just going through and doing some code reviews for notifications. I am not super knowledgable about the functionality there so I trust you 😃